source/introduction.txt

```diff
@@ -218,12 +176,20 @@ Traditionally developers query the database on the server and access the server
 from the client through a HTTP API. This requires more developer resources and lacks
 the flexibility of directly querying a database.
 
-App Services includes the :manual:`Query API </tutorial/query-documents>`,
-a powerful API that lets you access MongoDB Atlas data from the client.
-You can query Atlas like you're using a database driver and App Services enforces 
+App Services and the Realm SDKs include MongoDB Data Access,
+an API that lets you access MongoDB Atlas from the client using
+the :manual:`Query API </tutorial/query-documents>`.
+You can query Atlas like you're using a database driver and App Services enforces
 :ref:`data access rules <intro-web-schema>` rules for all requests.
 
-To learn more about the built-in MongoDB service, see :ref:`data-sources`.
+The following Realm SDKs let you query Atlas with MongoDB Data Access from client apps:
+
+- :ref:`Query MongoDB - Java SDK <java-mongodb-data-access>`
+- :ref:`Query MongoDB - .NET SDK <dotnet-mongodb-data-access>`
+- :ref:`Query MongoDB - Node.js SDK <node-mongodb-data-access>`
+- :ref:`Query MongoDB - React Native SDK <react-native-mongodb-data-access>`
+- :ref:`Query MongoDB - Swift SDK <ios-mongodb-data-access>`
+- :ref:`Query MongoDB - Web SDK <web-mongodb-data-access>`
```

Comment:
Confirming: No Flutter yet?

Reply:
correct, scheduled as part of their post GA epic